finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/Command.scala

```diff
@@ -422,8 +441,8 @@ class CommandCodec extends UnifiedProtocolCodec {
 
   val log = Logger(getClass)
 
-  val encode = new Encoder[Command] {
-    def encode(obj: Command) = Some(obj.toChannelBuffer)
+  val encode = new Encoder[RedisMessage] {
```

Comment:
Updated. I don't figure out what a 'RB_ID' is, so I did not add them, is that ok?